app/models/miq_report_result/result_set_operations.rb

```diff
@@ -45,7 +45,7 @@ def result_set_for_reporting(report_result, options)
         end
 
         result_set.map! { |x| x.slice(*report.col_order) }
-        result_set = result_set.stable_sort_by(sorting_columns, options[:sort_order])
+        result_set = result_set.tabular_sort(sorting_columns, options[:sort_order])
```

Comment:
this is used in api. Not sure if we need to test it on https://github.com/ManageIQ/manageiq-cross_repo maybe is enough to just run tests from this https://github.com/ManageIQ/manageiq-api/pull/547